docs/source/overview/delivery_services.rst

```diff
@@ -1023,3 +1023,20 @@ A text-based unique identifier for a Delivery Service. Many :ref:`to-api` endpoi
 .. [#cardinality] In source code and :ref:`to-api` responses, the "Long Description" fields of a Delivery Service are "0-indexed" - hence the names differing slightly from the ones displayed in user-friendly UIs.
 .. [#dupOrigin] These Delivery Services Types are vulnerable to what this writer likes to call the "Duplicate Origin Problem". This problem is tracked by :issue:`3537`.
 .. [#httpOnlyRegex] These regular expression types can only appear in the Match List of HTTP-:ref:`Routed <ds-types>` Delivery Services.
+
+
+Parameters
+----------------------
```

Comment:
nit: number of dashes should match the length of the header

Reply:
Changed